packages/expo-system-ui/ios/ExpoSystemUI/ExpoSystemUIModule.swift

```diff
@@ -3,22 +3,56 @@
 import ExpoModulesCore
 
 public class ExpoSystemUIModule: Module {
+  
+  public required init(appContext: AppContext) {
+    super.init(appContext: appContext)
+    
+    // TODO: Maybe read from the app manifest instead of from Info.plist.
+
+    // Set / reset the initial color on reload and app start.
+    let color = Bundle.main.object(forInfoDictionaryKey: "RCTRootViewBackgroundColor") as? Int
+    Self.setBackgroundColorAsync(color: color)
+  }
+  
   public func definition() -> ModuleDefinition {
     name("ExpoSystemUI")
 
     method("getBackgroundColorAsync") { () -> String? in
-      var color: String? = nil
-      EXUtilities.performSynchronously {
-        if let backgroundColor = self.appContext?.utilities?.currentViewController()?.view.backgroundColor?.cgColor {
-          color = EXUtilities.hexString(with: backgroundColor)
-        }
-      }
-      return color
+      Self.getBackgroundColor()
     }
 
     method("setBackgroundColorAsync") { (color: Int) in
```

Comment:
Since #14988 you can use `color: UIColor` and it'll be automatically converted 😊 (It's not supported for returned values yet)

Reply:
Well the Info.plist definition is in numeric format anyways.